stable/aws-vpc-cni/templates/daemonset.yaml

```diff
@@ -9,8 +9,12 @@ spec:
 {{ toYaml .Values.updateStrategy | indent 4 }}
   selector:
     matchLabels:
+{{- if .Values.originalMatchLabels }}
+      k8s-app: aws-node
+{{- else }}
```

Comment:
we could keep the extra labels and we don't have to exclude them.
it will help to have consistent labelling across workloads

Reply:
Hi @mohsen0 👋 
I am not sure I understand what you mean here?
The matchLabels field cannot be altered at all without recreating the ds, so it has to be just this single label.

Reply:
hey

So changing the labels on daemon set object will not cause a recreation, I just checked that by adding a label to a daemon set I have.

What I mean is having it like
```
{{- if .Values.originalMatchLabels }}
      k8s-app: aws-node
{{- end }}
      app.kubernetes.io/name: {{ include "aws-vpc-cni.name" . }}
      app.kubernetes.io/instance: {{ .Release.Name }}
```

so we can have a consistent naming in order to for example to look up logs in kibana or looking up resources.

Reply:
No worries! I thought you meant that for a minute, but then I thought you meant to modify the pod labels too, but that wouldn't work either! Hope you are well mate!